modules/data.land/tests/testthat/test-extract_soil_nc.R

```diff
@@ -0,0 +1,103 @@
+context("extract_soil_gssurgo")
+
+test_that("extract_soil_gssurgo returns valid NetCDF files for valid US coordinates", {
+  skip_on_cran()
+  skip_on_ci()
+  lat <- 40.1164
+  lon <- -88.2434
+  tmp_outdir <- withr::local_tempdir("gssurgo_test_")
+  
+  res <- extract_soil_gssurgo(
+    outdir = tmp_outdir, 
+    lat = lat,
+    lon = lon,
+    size = 2,
+    grid_size = 3,
+    grid_spacing = 100,
+    depths = c(0.15, 0.30)
+  )
+
+  expect_false(is.null(res))
+  
+  expect_type(res, "list")
+  expect_gt(length(res), 1)
+  expect_true(all(names(res) == "path"))
+  
+  # Validate files exist
+  file_paths <- unlist(res)
+  expect_true(all(file.exists(file_paths)))
+  
+  # Validate NetCDF content
+  if (requireNamespace("ncdf4", quietly = TRUE)) {
+    expected_vars <- c("fraction_of_sand_in_soil", "fraction_of_silt_in_soil", 
+                      "fraction_of_clay_in_soil", "soil_organic_carbon_stock")
+    
+    nc <- ncdf4::nc_open(file_paths[2]) # 2nd file because 1st file contains original gSSURGO data
```

Comment:
> 1st file contains original gSSURGO data

Can you rephrase this? I eventually figured out that it means the first ensemble member always uses the reported values without sampling from the estimated distributions, but it took me a while to figure that out.